stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -132,11 +132,18 @@ public static InspectorModulesProvider defaultInspectorModulesProvider(final Con
       @Override
       public Iterable<ChromeDevtoolsDomain> get() {
         ArrayList<ChromeDevtoolsDomain> modules = new ArrayList<ChromeDevtoolsDomain>();
-        modules.add(new Console());
         modules.add(new CSS());
         modules.add(new Debugger());
+        Runtime runtime = new Runtime();
+        modules.add(runtime);
         if (Build.VERSION.SDK_INT >= AndroidDOMConstants.MIN_API_LEVEL) {
-          modules.add(new DOM(new AndroidDOMProviderFactory((Application)context.getApplicationContext())));
+          DOM dom = new DOM(
+              new AndroidDOMProviderFactory(
+                  (Application)context.getApplicationContext()));
+          modules.add(dom);
+          modules.add(new Console(dom, runtime));
```

Comment:
I'm giving them access because this is a public API contract and I don't want to be discrete about what is needed for fear of having frequent and possibly confusing changes to the API.  There are solutions of course, but we should consider public API surface area and the fact that this is currently our only "configuration" mechanism we offer.